main-actions/src/main/scala/sbt/Package.scala

```diff
@@ -41,34 +40,38 @@ object Package {
     for ((key, value) <- mergeManifest.getEntries.asScala) {
       entryMap.get(key) match {
         case Some(attributes) => mergeAttributes(attributes, value)
-        case None             => entryMap put (key, value)
+        case None => entryMap put (key, value)
       }
     }
   }
 
-  final class Configuration(val sources: Seq[(File, String)], val jar: File, val options: Seq[PackageOption])
+  final class Configuration(val sources: Seq[(File, String)],
+                            val jar: File,
+                            val options: Seq[PackageOption])
   def apply(conf: Configuration, cacheStoreFactory: CacheStoreFactory, log: Logger): Unit = {
     val manifest = new Manifest
     val main = manifest.getMainAttributes
     for (option <- conf.options) {
       option match {
-        case JarManifest(mergeManifest)          => mergeManifests(manifest, mergeManifest)
-        case MainClass(mainClassName)            => main.put(Attributes.Name.MAIN_CLASS, mainClassName)
-        case ManifestAttributes(attributes @ _*) => main.asScala ++= attributes
-        case _                                   => log.warn("Ignored unknown package option " + option)
+        case JarManifest(mergeManifest) => mergeManifests(manifest, mergeManifest)
+        case MainClass(mainClassName) => main.put(Attributes.Name.MAIN_CLASS, mainClassName)
```

Comment:
Can we at least agree that `=>`s should be aligned? I find the former state to be so much more legible..